extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -38,7 +38,18 @@ const renderShoppingTagTemplate = (tagData) => (
         }
       ></span>
       <span class="amp-story-shopping-tag-pill-text">
-        {tagData['product-tag-text'] || '$' + tagData['product-price']}
+        {(tagData['product-tag-text'] && (
+          <span class="amp-story-shopping-product-tag-text">
```

Comment:
Please remove the `amp-story-shopping-product-tag-text` span wrapper. I think that is part of a different PR.